homeassistant/components/google/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  config-flow:
+    status: done
+    comment: Integration uses an OAuth config flow.
+  brands: done
+  dependency-transparency: todo
+  common-modules: done
+  has-entity-name: done
+  action-setup:
+    status: todo
+    comment: |
+      Actions are current setup in `async_setup_entry` and need to be moved
+      to `async_setup`.
```

Comment:
We should probably also start removing this and moving it to the config_entry only schema
`CONFIG_SCHEMA = vol.Schema(cv.removed(DOMAIN), extra=vol.ALLOW_EXTRA)`

Reply:
What functionality is this allowing now? I'm actually not sure what it allows vs if this is just dead code cleanup.